Makefile

```diff
@@ -310,19 +313,16 @@ install_cert_manager:
 	kubectl apply --validate=false -f https://github.com/jetstack/cert-manager/releases/download/v1.0.4/cert-manager.yaml
 
 _kustomize:
```

Comment:
Since it's only generate-deployment.sh who needs kustomize https://github.com/devfile/devworkspace-operator/blob/main/Makefile#L185, maybe it makes sense to move installing there ?

Reply:
Good idea -- done.